edit-post/README.md

```diff
@@ -11,11 +11,15 @@ They can be found in the global variable `wp.editPost` when defining `wp-edit-po
 
 Experimental components can be found under `wp.editPost.__experimental`. Experimental components are still being evaluated and can change in a future version.
 
-### PluginSidebar (Experimental)
+### PluginSidebar
+**Experimental**
 
 Renders a sidebar when activated. The contents within the `PluginSidebar` will appear as content within the sidebar.
 
-If you wish to display the sidebar, you can with use the [`PluginMoreMenuItem`](#pluginmoremenuitem-experimental) component or the `wp.data.dispatch` API.
+If you wish to display the sidebar, you can with use the [`PluginMoreMenuItem`](#pluginmoremenuitem) component or the `wp.data.dispatch` API:
+```js
+wp.data.dispatch( 'core/edit-post' ).openGeneralSidebar( 'plugin-name/sidebar-name' );
```

Comment:
Since the `PluginMoreMenuItem` is generic with a target type and a target name, we should provide a generic action as well

`wp.data.dispatch( 'core/edit-post' ).openPluginTarget( 'sidebar', 'plugin-name/sidebar-name' );`

Reply:
That's why I don't like general APIs that have a different behavior based on the provided params.

Having a generalized solution also triggers another question if this should be a part of `edit-post` or `plugin`? The same applies to `PluginMoreMenuItem`.

Reply:
Why are you saying different behavior? Can't we assume any target can be opened/closed. 

And moving to `plugin` module might be a good option.

Other scripts would "register" targets :)